docs/spec/spec.md

```diff
@@ -1820,6 +1876,28 @@ service on new graphql:Listener(9090) {
 }
 ```
 
+#### 7.2.3 Field-level Cache Configuration
+
+The `cacheConfig` field is used to provide the field-level cache configs. The fields are as same as the operation cache configs.
```

Comment:
Let's add an anchor here as well. Better to link to the specific section, instead of linking to the top-level cache topic.